loops.js

```diff
@@ -16,9 +19,19 @@
 // loop...
 // loop...
 // klaar!
- // ==========================================
+// ==========================================
 
+// for loop log 4 times "loop..."
+for (let i = 0; i < 5; i++) {
+    if (i < 4) {
+        console.log("loop...");
+    }
 
+    // log the 5th loop 1 time "klaar"
+    if (i === 4) {
+        console.log("klaar!")
+    }
+}
```

Comment:
je 2e if is onnodig aangezien het alles boven de 4 is

for (let i = 0; i < 5; i++) {
  if (i === 4) {
    console.log('klaar!')
  } else {
    console.log('loop...');
  }
}